### PR TITLE
Fixing the console-extension route in free-int

### PR DESCRIPTION
**Pull request by dak1n1**
* Use service-signer cert for route
* Use oc_route for idempotent reencrypt route creation with Ops certs

This also contains changes from https://github.com/openshift/online-console-extensions/pull/17

### CODE REVIEW

File: ansible/roles/oso_console_extensions/tasks/main.yml [hunk 1]
```diff
@@ -52,3 +46,15 @@
...
 - debug: var=start_build_out
+
+- name: Create edge route for console-extensions url
+  oc_route:
+    name: "{{ osce_appname }}"
+    namespace: "{{ osce_namespace }}"
+    service_name: "{{ osce_appname }}"
+    tls_termination: edge
+    port: 8080
+    host: oso-console-extensions-openshift-infra.d800.free-int.openshiftapps.com
```
Comment (damemi) [inline ansible/roles/oso_console_extensions/tasks/main.yml]: could this be dynamic so that it won't break devenvs? or should we table that for now

Comment (dak1n1) [reply ansible/roles/oso_console_extensions/tasks/main.yml]: yes, I need to find the param to pass for that one. I'll update that

**Comment by dak1n1** [comment]
The url is now configurable and we'll have to pass it in from the devenvs

**Comment by dak1n1** [comment]
(It's already passed in on the Ops side)

**Review by damemi — APPROVED** [review]

**Pull request merged**